plugins/modules/aci_access_port_block_to_access_port.py

```diff
@@ -381,26 +405,47 @@ def main():
     type_port = module.params.get("type")
 
     aci = ACIModule(module)
+
     aci_class = "infraAccPortP"
     aci_rn = "accportprof"
     if type_port == "fex":
         aci_class = "infraFexP"
         aci_rn = "fexprof"
-    aci.construct_url(
-        root_class=dict(
-            aci_class=aci_class,
-            aci_rn="infra/" + aci_rn + "-{0}".format(interface_profile),
+    subclass_1 = dict(
+        aci_class=aci_class,
+        aci_rn="{0}-{1}".format(aci_rn, interface_profile),
+        module_object=interface_profile,
+        target_filter={"name": interface_profile},
+    )
+    # NOTE: normal rn: hports-{name}-typ-{type}, hence here hardcoded to range for purposes of module
```

Comment:
what do you mean exactly with this comment? this module never needs the other type input option?

Reply:
This is an old comment, the previous author made that I think is not needed. I forgot to remove it.